src/main/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/DraftController.java

```diff
@@ -79,10 +80,12 @@ public Draft read(
     })
     public DraftList readAll(
         @RequestHeader(AUTHORIZATION) String authHeader,
-        @RequestHeader(SERVICE_HEADER) String serviceHeader
+        @RequestHeader(SERVICE_HEADER) String serviceHeader,
+        @RequestParam(name = "after", required = false) Integer after,
+        @RequestParam(name = "limit", required = false, defaultValue = "10") int limit
```

Comment:
potentially a new version of the API needed for this?

Reply:
we can change it to return everything if `limit` query param is not added